dbms/tests/queries/0_stateless/00921_datetime64_basic.sql

```diff
@@ -0,0 +1,19 @@
+USE test;
+
+DROP TABLE IF EXISTS A;
+
+SELECT CAST(1 as DateTime64('abc')); -- { serverError 43 } # Invalid scale parameter type
+SELECT CAST(1 as DateTime64(100)); -- { serverError 69 } # too big scale
+SELECT CAST(1 as DateTime64(-1)); -- { serverError 43 } # signed scale parameter type
```

Comment:
Here is the test cases for invalid scale values

Reply:
Yep. Just haven't noticed it before.